### DISCUSSION TITLE
ERR_CONTENT_DECODING_FAILED when deployed in Lambda

### DISCUSSION THREAD

**Discussion by jjun3188** [graphql/graphiql#1922] [Q&A / Support]
Hi.

I am getting a blank HTML page with `net::ERR_CONTENT_DECODING_FAILED 200` in the browser console when I open GraphiQL.

It works just fine when I run the application locally, but I get that error when deployed in Lambda.

Is anyone having the same issue?

**Comment by acao** [comment]
converted to a discussion as this is an implementation support question and not an issue with our project. hopefully someone else has come across a similar issue and can help you!
